modules/ossm-cr-mixer.adoc

```diff
@@ -5,7 +5,7 @@
 [id="ossm-cr-mixer_{context}"]
 = Istio Mixer example
 
-Here is an example that illustrates the Mixer parameters for the {ProductName} custom resource and a description of the available parameters with appropriate values.
+Here is an example that illustrates the Mixer parameters for the `ServiceMeshControlPlane` and a description of the available parameters with appropriate values.
 
 [source,yaml]
 ----
```

Comment:
Most lines in this example are rendered in red because the indentation is wrong. `policy` and `telemetry` should be at the same level as `enabled`.

Reply:
Fixed.